app/src/main/java/org/mozilla/fenix/components/FenixSnackbar.kt

```diff
@@ -157,8 +162,20 @@ class FenixSnackbarPresenter(
         actionName: String? = null,
         isError: Boolean = false
     ) {
-        FenixSnackbar.make(view, length, isError).setText(text).let {
+        val snackbar = FenixSnackbar.make(view, length, isError).setText(text).let {
             if (action != null && actionName != null) it.setAction(actionName, action) else it
-        }.show()
+        }
+
+        if (view.context.settings().shouldUseBottomToolbar) {
+            snackbar.view.setPadding(
+                0,
+                0,
+                0,
+                view.context.resources.getDimensionPixelSize(R.dimen.browser_toolbar_height
+                )
+            )
+        }
+
+        snackbar.show()
```

Comment:
If you agree with my above two comments, it might wind up looking something like this.

```kotlin
) {
    val shouldUseBottomToolbar = view.context.settings().shouldUseBottomToolbar
    val toolbarHeight = view.context.resources
        .getDimensionPixelSize(R.dimen.browser_toolbar_height)

    FenixSnackbar.make(view, length, isError).apply {
        setText(text)
        if (action != null && actionName != null) setAction(actionName, action)
        view.setPadding(
            0,
            0,
            0,
            if (shouldUseBottomToolbar) toolbarHeight else 0
        )
        show()
    }
}
```

Reply:
This feels a bit cleaner, thanks!